### PR TITLE
Update Packages

### PR DESCRIPTION
**Pull request by nightfuryninja**
Hello, I have finally got around to fixing the package issue where it would not be able to find the SteamAuth.dll . This should now work (hopefully). I originally made a mistake in updating the wrong number so here is a working one. I have tested this and so far it is working. (Some warning to do with Mono) but other than that it all works.

### CODE REVIEW

File: SteamBot/ExampleBot.csproj [hunk 1]
```diff
@@ -47,9 +47,8 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\packages\protobuf-net.2.0.0.668\lib\net40\protobuf-net.dll</HintPath>
     </Reference>
-    <Reference Include="SteamAuth, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+    <Reference Include="SteamAuth, Version=1.1.0.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (BlueRaja) [inline SteamBot/ExampleBot.csproj]: Reverting to the old version?  Wasn't the whole point to upgrade to the latest?

Comment (nightfuryninja) [reply SteamBot/ExampleBot.csproj]: That isn't reverting it to the old one. The .dll referenced is still 2.0.0 . When I built that new file, I removed the SteamAuth reference completely and re added it. This is how it should be (according to visual studio) even though the .dll version is 2.0.0 and that says 1.1.0.0.

Comment (nightfuryninja) [reply SteamBot/ExampleBot.csproj]:
```
<HintPath>..\packages\SteamAuth.2.0.0\lib\net45\SteamAuth.dll</HintPath>
```
The .dll is 2.0.0 but the version is not? I don't know why but that is the fix.

Comment (BlueRaja) [reply SteamBot/ExampleBot.csproj]: I think the AssemblyInfo.cs was not updated in SteamAuth.  Could you do that and push a new version?

Comment (nightfuryninja) [reply SteamBot/ExampleBot.csproj]: I changed the AssemblyInfo.cs and I pushed a new version to Nuget but in the current email that I have sent you, I keep getting the targeted framework error. I do not understand why though.

**Pull request closed without merging**